pom.xml

```diff
@@ -50,7 +50,7 @@
     <site>
```

Comment:
I tried to find a way to remove this from the pom.xml and instead use some command line parameter.  Unfortunately, it seems that mvn site:stage requires <site> to be specified to figure out links between submodules, even if a link is provided.